### PR TITLE
Uses LAN and USB flags properly

### PR DESCRIPTION
**Pull request by johnnyman727**
Turns out this was the root cause of #358 and #238. Probably more issues too, they just weren't reported.

### CODE REVIEW

File: lib/controller.js [hunk 1]
```diff
@@ -68,7 +72,7 @@ Tessel.list = function(opts) {
             // Report that selected Tessel to the user
             logs.info('Multiple Tessels found.');
             if (tessel) {
-              logs.info('Will default to', tessel.name, '.');
+              logs.info('Will default to', tessel.name + '.');
```
Comment (johnnyman727) [inline lib/controller.js]: This was bugging me because it was print out as: `Will default to Frank .` with a space before the period.

Comment (LinusU) [inline lib/controller.js]: Now you use both `,` and `+` on the same line

Comment (johnnyman727) [inline lib/controller.js]: Yeah, perhaps `sprintf` is the reasonable way to do this?

Comment (LinusU) [inline lib/controller.js]: This should work and looks nice:

```javascript
logs.info('Will default to %s.', tessel.name);
```

It uses `util.format` which is a builtin in Node that does this. That works with `console.log` as well :+1:

**Comment by johnnyman727** [comment]
@LinusU since you already started, do you want to give it a full review?

**Comment by LinusU** [comment]
Already on it ;)

**Comment by LinusU** [comment]
Code looks good, tests pass locally and on CI.

The only thing is the one comment I made:

```diff
-              logs.info('Will default to', tessel.name + '.');
+              logs.info('Will default to %s.', tessel.name);
```

LGTM :+1:

**Comment by johnnyman727** [comment]
Awesome, thanks a million! Will make that change and merge.

**Pull request merged**